### ISSUE TITLE
Doesn't work with BBC iPlayer

### ISSUE THREAD

**Issue by pisss** [ettoolong/MaximizeVideo#53]
Tested both manual and auto. Seems to be running something called "SMPToucan".

Not entirely sure how this all works but googling found me this, which seems relevant as it is also struggling with the players being hidden with different tags: https://bugzilla.mozilla.org/show_bug.cgi?id=1762042

**Comment by Wallboy** [comment]
Came here to say the same thing. Doesn't detect the BBC iPlayer.

**Comment by JAYZ303** [comment]
Yup, it's the only thing I consistently use that it doesn't support.

**Comment by ettoolong** [comment]
MaximizeVideo version 0.0.23 for Firefox and Chrome now supports Shadow DOM. 🙂

**Comment by pisss** [comment]
hell yeah! thank you so much
